src/usb/uf2/ghostfat.c

```diff
@@ -518,5 +561,6 @@ int write_block (uint32_t block_no, uint8_t *data, WriteState *state)
     }
   }
 
-  return 512;
+  STATIC_ASSERT(BPB_SECTOR_SIZE == 512); // if sector size changes, may need to re-validate this code
```

Comment:
You are right, that it exists elsewhere.
I had it here because I am unsure if the caller expects 512, or if the caller can accept 4k result.

The English saying, "belt and suspenders" applies... you could always wear only one, but wearing both belt and suspenders makes it even less likely your pants will fall.

Again, if you feel strongly, you can edit this branch directly before accepting this PR.  I will not stop you.

However, again because it has zero impact on final code, it is my opinion that the cost of an extra line of text is worthwhile.

Reply:
Yeah, I feel strongly, but I am even more lazy. I will leave it as it is now and will **clean up** later on when I work with bootloader.